features/public-form/ui/test-submission-badge.module.css

```
@@ -0,0 +1,35 @@
+  right: 0.5rem;
+  z-index: 1000;
+  outline: none;
+}
+
+.badge {
+  position: static;
+  bottom: auto;
+  opacity: 1;
+}
+
+.badge :global(.sd-slider__tooltip-panel)::after {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)